### ISSUE TITLE
Does training user learning_rate decay?

### ISSUE THREAD

**Issue by renmada** [facebookresearch/mae#22]
Code set warm up epochs as 40, but lr scheduler does not show up in code

**Comment by KaimingHe** [comment]
See https://github.com/facebookresearch/mae/blob/main/util/lr_sched.py

**Comment by renmada** [comment]
> See https://github.com/facebookresearch/mae/blob/main/util/lr_sched.py

I mean lr scheduler does not show up in "main_pretrain.py"

**Comment by KaimingHe** [comment]
The lr adjust happens here: https://github.com/facebookresearch/mae/blob/main/engine_pretrain.py#L43
